### PR TITLE
refactor: prevent duplicate selection

### PR DESCRIPTION
**Pull request by dasarathg68**
# Description
Prevent duplicate selection of founders and members in the CreateOfficerTeam form.

## Contribution

For your PR please add a commnent to eache file edited to explain the changes you made.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist

Make sur you run the following commands before submitting your PR:

- /app

```bash
npm run build
npm run test:unit
npm run type-check
npm run lint
npm run format
```

- /backend

```bash
npm run build
npm run test
npm run lint
npm run format
```

- /contract

```bash
npm run test
npm run lint
npm run format
```

- **Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines

### CODE REVIEW

File: app/src/components/forms/CreateOfficerTeam.vue
Comment (dasarathg68) [inline app/src/components/forms/CreateOfficerTeam.vue:1]: Prevent the duplicate selection of founders and members in the CreateOfficerForm component.

**Comment by hermannleboss** [comment]
When there is a value in the name input, and i'm updating the address input it's searching a mach for both. 

expected behaviour:

I have 2 user 

Hermann 0x123
Dasarath 0xABC

if i have in the input of name Da
and in the input of address 0x

Normaly the result of the search should be the latest update. 

That mean if i'm typing 0x1... i should have address search 
and if i'm typing Da.. i should have name search.

image

**Comment by hermannleboss** [comment]
Second if i click outside of an input the search result should not be visible. 

image

**Comment by dasarathg68** [comment]
@hermannleboss 
The two suggestions are beyond the scope of this PR, I will create issues for them and address them separately.

**Comment by hermannleboss** [comment]
> @hermannleboss The two suggestions are beyond the scope of this PR, I will create issues for them and address them separately.

OK

**Pull request merged**